src/mz/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "mz"
+version = "0.1.0"
+edition = "2021"
```

Comment:
```suggestion
[package]
name = "mz"
name = "mz-adapter"
description = "Materialize's CLI."
version = "0.0.0"
edition = "2021"
rust-version = "1.62.0"
publish = false
```

Reply:
Done! I should have looked at Materialize code before choosing Actix. At first, I was in doubt about which one to pick and ended up with the most "used".